### PR TITLE
1496 read company data loans

### PR DESCRIPTION
**Pull request by jacobvjk**
closes ADO 1496: https://dev.azure.com/2DegreesInvesting/2DegreesInvesting/_workitems/edit/1496

This PR:
- reads in the prewargled stress test master data into the loan book workflow
- adapts the corresponding utils and read_company_data functions to enable that

### CODE REVIEW

File: stress_test_model_loan_book.R [hunk 1]
```diff
@@ -121,7 +124,7 @@ scenarios <- c(
   "B2DS",
   "CPS",
   "NPS",
-  "NPSRTS",
+  # "NPSRTS",
```
Comment (MirijaHa) [inline stress_test_model_loan_book.R:127]: I would prefer removing this instead of commenting it out

Comment (jacobvjk) [reply stress_test_model_loan_book.R]: I concur

**Review by MirijaHa — COMMENTED** [review]
Just to go sure,  the scope of the ticket is only to read the dataset in? Asking since some prep steps and the inclusion in calculations are not yet happening?

**Comment by jacobvjk** [comment]
yes, this is the first sub task

**Comment by jacobvjk** [comment]
the rest will be done in one joint PR next. I wanted to do this separately, as the master data loans file was also not prewrangeld yet

**Review by MirijaHa — APPROVED** [review]
LGTM

**Pull request merged**